src/sequence_statistics.py

```diff
@@ -0,0 +1,229 @@
+import requests
+import random
+import json
+
+
+# API_URL = 'http://0.0.0.0:5000'
+API_URL = 'http://127.0.0.1:5001'
+
+
+# -------------- Fonctions pour calculer les stats sur les syllabes ------------------------- #
+
+
+# Ici, pour un mot donné, on incrémente les statistiques de ses syllabes adjacentes
+def get_statistics(language,word):
```

Comment:
this function is quite long and hence difficult to read, it would be better to split it into consistent blocks.